virttest/qemu_vm.py

```diff
@@ -996,6 +997,24 @@ def add_memorys(devices, params):
                 for name in params.objects("mem_devs"):
                     dev = devices.memory_define_by_params(params, name)
                     devs.extend(dev)
+            machine_dev = devices.get_by_properties({"type": "machine"})[0]
+            machine_cmd = machine_dev.cmdline_nd()
+            output = re.findall(r",memory-backend=mem-([\w|-]+)", machine_cmd)
+            if output:
+                name = output[0]
+                backend_options = {}
+                backend_options["size"] = "%sM" % params["mem"]
+                if params.get("hugepage_path"):
+                    backend_options["backend"] = "memory-backend-file"
+                    backend_options["mem-path"] = params["hugepage_path"]
+                backend_param = Params(backend_options)
+                dev = devices.memory_object_define_by_params(backend_param,
+                                                             name)
+                devs.append(dev)
+            elif params.get("hugepage_path"):
+                cmd = "-mem-path %s" % params["hugepage_path"]
+                devs.append(StrDev('extra', cmdline=cmd))
```

Comment:
Suggest the name `mem-path` instead of `extra`

Reply:
Thanks, updated.  
Test passed on s390 host. 
@luckyh @yanan-fu Pls help review, thanks.